Makefile

```diff
@@ -56,11 +56,17 @@ macros/src/vector.rs: src/devices/*/interrupt.rs
 	@echo -e "\tGEN-VECTOR\t>macros/src/vector.rs"
 	@./gen-intr-lut.sh $^ >$@
 
+check: all
+	@echo -e "\tCHECK"
+	@cargo check --all-features
+
```

Comment:
I'd rather keep the makefile just for generating the files, not for actually invoking any kind of compilation, linting, etc.

Reply:
That's a fair point.

I only added it after forgetting to pass `--all-features` and hitting the compile warning one too many times.

Reply:
It's gone.